<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>911</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="TabSubstitute">
     <attribute name="title">
      <string>Substitution Cipher</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Plain Text</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>421</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Transpose Key</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>421</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cipher Key</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>210</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Encrypt">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Decrypt">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>320</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TabCeasar">
     <attribute name="title">
      <string>Ceasar Cipher</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>39</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Plain Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Cipher Text</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ceasarDecrypt">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>310</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxRotate">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>70</y>
        <width>126</width>
        <height>66</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-256</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>90</y>
        <width>117</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TabReverse">
     <attribute name="title">
      <string>Reverse Word</string>
     </attribute>
     <widget class="QLabel" name="word">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>117</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Word:</string>
      </property>
     </widget>
     <widget class="QLabel" name="word_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>117</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Reversed:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="originalWord">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>340</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="wordReversed">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>340</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reverseButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>240</width>
        <height>54</height>
       </rect>
      </property>
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>240</width>
        <height>54</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>321</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="text">
     <string>Adam Smith</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
